extensions/sql-database-projects/src/dialogs/updateProjectFromDatabaseDialog.ts

```diff
@@ -0,0 +1,610 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
why was this added? We usually try to avoid disabling the lint rules

Reply:
Looks like something leftover from the initial PR.  Removed